tests/renderer/components/__snapshots__/settings-general-gist-spec.tsx.snap

```diff
@@ -0,0 +1,27 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Looks like this snapshot is outdated, probably this branch existed before the Vitest migration. Can you nuke it and update the snapshot?